x/stakeibc/keeper/keeper.go

```diff
@@ -106,24 +107,24 @@ func (k *Keeper) SetHooks(gh types.StakeIBCHooks) *Keeper {
 	return k
 }
 
-func (k Keeper) GetChainID(ctx sdk.Context, connectionID string) (string, error) {
-	conn, found := k.IBCKeeper.ConnectionKeeper.GetConnection(ctx, connectionID)
+// GetAuthority returns the x/icaoracle module's authority.
```

Comment:
icaoracle module's authority? Should this be stakeibc module's authority?

Reply:
ah yeah, good catch! done